modules/friendlywelcome.js

```diff
@@ -598,6 +598,46 @@ Twinkle.welcome.templates = {
 			'welcomeen-uk': {
 				description: 'welcome for users whose first language appears to be Ukrainian',
 				syntax: '{{subst:welcomeen-uk}}'
+			},
```

Comment:
It looks like the other language welcome templates are alphabetized by the human-readable language. For example, welcomeen-zh's human-readable language is "Chinese", and it is listed towards the top of the list.

Can we please also alphabetize these new templates?